model/manifest.go

```diff
@@ -14,11 +14,12 @@ import (
 )
 
 const (
-	PLUGIN_CONFIG_TYPE_TEXT      = "text"
-	PLUGIN_CONFIG_TYPE_BOOL      = "bool"
-	PLUGIN_CONFIG_TYPE_RADIO     = "radio"
-	PLUGIN_CONFIG_TYPE_DROPDOWN  = "dropdown"
-	PLUGIN_CONFIG_TYPE_GENERATED = "generated"
+	PLUGIN_CONFIG_TYPE_TEXT              = "text"
+	PLUGIN_CONFIG_TYPE_BOOL              = "bool"
+	PLUGIN_CONFIG_TYPE_RADIO             = "radio"
+	PLUGIN_CONFIG_TYPE_DROPDOWN          = "dropdown"
+	PLUGIN_CONFIG_TYPE_GENERATED         = "generated"
+	PLUGIN_CONFIG_TYPE_USER_AUTOCOMPLETE = "user_autocomplete"
```

Comment:
How about just making this "username" instead? That would convey that the value is a username as opposed to a user id, and I don't think "_autocomplete" is really necessary. Just thinking about other setting schema schemas that I've used, such as [for CloudFormation](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/parameters-section-structure.html#aws-specific-parameter-types), I think it's kind of implicit that if you specify a type as specific as "username", you're gonna get the most appropriate UI for it.

Reply:
I'm 0/5, I'll make the update